.ci/jobs/dist-tag-mbp.yml

```diff
@@ -0,0 +1,42 @@
+---
+- job:
+    name: apm-agent-rum/dist-tag-mbp
+    display-name: Dist tag a version for Elastic Synthetics
+    description: Dist tag a version for Elastic Synthetics.
+    project-type: multibranch
+    script-path: .ci/dist-tag.groovy
+    scm:
+      - github:
+          branch-discovery: no-pr
+          discover-pr-forks-strategy: merge-current
+          discover-pr-forks-trust: permission
+          discover-pr-origin: merge-current
+          discover-tags: false
+          disable-pr-notifications: true
+          head-filter-regex: '^main$'
```

Comment:
I'd say to add this PR, so you can test your changes if the other PR is related to the JJBB

Reply:
I will use this PR to define the Jenkins job pipeline and move the jjbb jenkins job definition in another PR.